uui/src/lib/List/List.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+  import type { ListProps } from './List.types';
+  import Box from '../Box/box.svelte';
+  import { current_component } from 'svelte/internal';
+
+  // Props
+  type $$Props = ListProps;
+  export let disablePadding: boolean | undefined = undefined;
+</script>
+
+<Box
+  element="ul"
+  eventComponent={current_component}
+  {...$$restProps}
+  class="mdc-deprecated-list {$$restProps.class || ''} {disablePadding ? 'disabled-padding' : ''}"
```

Comment:
Lets add "list" as  a class just for continuity. I want to move away from mdc for the users (it should still be there just want to alleviate the need for users to use the mdc classes)